BurnOutSharp/ProtectionType/PlayJ.cs

```diff
@@ -0,0 +1,89 @@
+    /// The file format itself appears to be an encrypted MP3 (https://www.wired.com/2000/03/ads-take-aim-at-online-music/).
+    /// The IANA assignment for the PlayJ format: https://www.iana.org/assignments/media-types/audio/vnd.everad.plj
+    /// The official player has been archived and can still be used to listen to the music (https://web.archive.org/web/20010417025347/http://dlp.playj.com:80/playj/PlayJIns266.exe).
+    /// The music files themselves do not contain advertisements, which is instead handled on the side of the music players and plugins capable of playing the format.
+    /// According to the ReadMe.txt found in https://web.archive.org/web/20010417025347/http://dlp.playj.com:80/playj/PlayJIns266.exe, there's 2 distinct versions of the format. The differences between versions are currently unknown.
+    /// There are a few sources of PlayJ music archived online, these being a few examples:
+    /// https://web.archive.org/web/*/http://dlp.playj.com/*/
+    /// https://web.archive.org/web/*/http://brightnet.music.tucows.com/*/
+    /// Also noted in the ReadMe.txt present in https://web.archive.org/web/20010417025347/http://dlp.playj.com:80/playj/PlayJIns266.exe as hosting PlayJ music:
+    /// http://www.listen.com
+    /// http://www.launch.com
+    /// Further information and resources:
+    /// https://www.entrepreneur.com/business-news/sales-marketing-playing-around/33284
+    /// https://www.adweek.com/brand-marketing/j-records-and-everad-ink-pact-47882/
+    /// https://adage.com/article/news/everad-s-playj-partners-clive-davis-j-records/11043
+    /// https://www.nytimes.com/2000/07/07/technology/search-for-digital-music-revenue-takes-new-direction.html
+    /// </summary>
+    public class PlayJ : IPathCheck, IPortableExecutableCheck
+    {
+        public string CheckPortableExecutable(string file, PortableExecutable pex, bool includeDebug)
+        {
+            // Get the sections from the executable, if possible
+            var sections = pex?.SectionTable;
+            if (sections == null)
+                return null;
+
+            // Found in "PlayJ.exe" (https://web.archive.org/web/20010417025347/http://dlp.playj.com:80/playj/PlayJIns266.exe) and "CACTUSPJ.exe" ("Volumia!" by Puur (Barcode 7 43218 63282 2) (Discogs Release Code [r795427])).
+            string name = pex.FileDescription;
+            if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("PlayJ Music Player", StringComparison.OrdinalIgnoreCase))
+                return $"PlayJ Music Player";
+            // Found in "PJSTREAM.DLL" "Volumia!" by Puur (Barcode 7 43218 63282 2) (Discogs Release Code [r795427]).
+            name = pex.FileDescription;
+            if (!string.IsNullOrWhiteSpace(name) && name.StartsWith("EVAUX32 Module", StringComparison.OrdinalIgnoreCase))
```

Comment:
I'm a tad suspicious of this particular description.

Reply:
Ah yep, forgot a pair of parentheses.

Reply:
Sorry for the vagueness, I didn't mean the comment string, I meant the literal `FileDescription` value of "EVAUX32 Module". That doesn't scream out to me that it's PlayJ, at least from what I see.

Reply:
Cleared up offline. There's no reference to this outside of PlayJ.